cmd/otel-agent/subcommands/run/command.go

```diff
@@ -185,10 +188,13 @@ func runOTelAgentCommand(ctx context.Context, params *subcommands.GlobalParams,
 			return configsyncimpl.NewParams(params.SyncTimeout, params.SyncDelay, true)
 		}),
 
-		fx.Provide(func() tagger.Params {
-			return tagger.NewNodeRemoteTaggerParamsWithFallback()
+		remoteTaggerFx.Module(tagger.RemoteParams{
+			RemoteTarget: func(c coreconfig.Component) (string, error) { return fmt.Sprintf(":%v", c.GetInt("cmd_port")), nil },
```

Comment:
Would the remote target not always be `cmd_port` ? Do we need an option to change it ?